schemas/1.3.0/adaptive-card.json

```diff
@@ -982,19 +938,19 @@
         },
         "max": {
           "type": "string",
-          "description": "Hint of maximum value expressed in ISO-8601 format (may be ignored by some clients)."
+          "description": "Hint of maximum value expressed in YYYY-MM-DD(may be ignored by some clients)."
         },
         "min": {
           "type": "string",
-          "description": "Hint of minimum value expressed in ISO-8601 format (may be ignored by some clients)."
+          "description": "Hint of minimum value expressed in YYYY-MM-DD(may be ignored by some clients)."
         },
         "placeholder": {
           "type": "string",
           "description": "Description of the input desired. Displayed when no selection has been made."
         },
         "value": {
           "type": "string",
-          "description": "The initial value for this field expressed in ISO-8601 format."
+          "description": "The initial value for this field expressed in YYYY-MM-DD."
```

Comment:
This should indeed be ISO8601. It has to support a time offset, otherwise it will not work properly across time zones.

Reply:
Should we remove "may be ignored by some clients"?

Reply:
Good point.

---
In reply to: [447780023](https://github.com/microsoft/AdaptiveCards/pull/4255#discussion_r447780023) [](ancestors = 447780023)

Reply:
Thinking about it, we may still want to document that this is ignored on earlier versions. How about "(may be ignored by some clients prior to version 1.3)"?

---
In reply to: [448481109](https://github.com/microsoft/AdaptiveCards/pull/4255#discussion_r448481109) [](ancestors = 448481109,447780023)